doc/users/next_whats_new/engformatter_usetex_usemathtext.rst

```diff
@@ -0,0 +1,11 @@
+`EngFormatter` now accepts `usetex`, `useMathText` as keyword only arguments
+``````````````````````````````````````````````````````````````````````````````````````````````
+
+A public API has been added to `EngFormatter` to control how the numbers in the ticklabels will be rendered.
+By default, `useMathText` evaluates to `rcParams['axes.formatter.use_mathtext']` and
```

Comment:
```suggestion
By default, ``useMathText`` evaluates to ``rcParams['axes.formatter.use_mathtext']`` and
```

Reply:
Use double-backticks for literal code. Single backticks are for sphinx roles. Holds also for all the following backtick uses in this file.